ibu-imager/seedcleanup/seedcleanup.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package seedcleanup
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-kni/lifecycle-agent/ibu-imager/ops"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// Default location for etcdStaticPodFile
+	etcdStaticPodFile      = "/etc/kubernetes/manifests/etcd-pod.yaml"
+	etcdStaticPodContainer = "etcd"
+
+	containerListDoneFile = "/var/tmp/container_list.done"
+
+	// Default location for backing up seed cluster certificates
+	backupCertsDir = "/var/tmp/backupCertsDir"
+)
+
+// SeedCleaner handles cleanup operations after seeding, removing temporary files
+// and executing additional cleanup steps as needed.
+type SeedCleaner struct {
+	log                  *logrus.Logger
+	ops                  ops.Ops
+	backupDir            string
+	containerRegistry    string
+	authFile             string
+	recertContainerImage string
+	recertSkipValidation bool
+}
+
+// NewSeedCleaner is a constructor function for SeedCleaner
```

Comment:
agree, but one of the linters in this project asks for a comment, otherwise complains
```
hack/lint.sh
./ibu-imager/seedcleanup/seedcleanup.go:50:1: exported function NewSeedCleaner should have comment or be unexported
Found 1 lint suggestions; failing.
make: *** [Makefile:109: lint] Error 1
```

Reply:
IMO let's get rid of the linter then, it's ridiculous

Reply:
cc/ @jc-rh @donpenney

Reply:
agree with @omertuc

Reply:
k, thanks. Added #88 for doing that.